### PR TITLE
support pip install

### LINKED ISSUE #320: add releases to PyPi
It would be nice to have a streamlined way of uploading releases to PyPi.

### PR DESCRIPTION
**Pull request by rc**
Closes #320.

### CODE REVIEW

**Comment by rc** [comment]
Hi @vlukes, could you have a look? You can try a `TestPyPI` version using:
```
      python3 -m venv venv
      source venv/bin/activate
      python3 -m pip install -U -i https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple/ sfepy
      python3 -m pip install pytest
      python3 -c "import sfepy; sfepy.test('-v', '--durations=0')"
      deactivate
      # rm -rf venv
```
If that works for you, I will put 2022.2.1 on PyPI for testing it on the real site.

See also the updated `doc/release_tasks.rst` and `doc/installation.rst`.

**Comment by vlukes** [comment]
It works for me. Tests pass except `test_spregion2d()`.

`
1 failed, 199 passed, 10 warnings in 342.78s (0:05:42)
`

File: doc/installation.rst [hunk 1]
```diff
@@ -112,20 +54,25 @@ using Anaconda and Windows 8.1+ using Anaconda.
...
 - `Anaconda`_ distribution: install `sfepy` from `conda-forge`_ channel::
 
-   conda install -c conda-forge sfepy
+    conda install -c conda-forge sfepy
 
-- Debian/(K)Ubuntu: install `python-sfepy`::
+  See `Notes on Multi-platform Python Distributions`_ for additional notes.
 
-   sudo apt-get install python-sfepy
+- Debian/(K)Ubuntu: install `python3-sfepy`::
```
Comment (vlukes) [inline doc/installation.rst:70]: Not available for [20.04 LTS focal](https://packages.ubuntu.com/focal/)

**Comment by rc** [comment]
I have simplified the docs - removed the distribution specific info.

**Pull request merged**